### DISCUSSION TITLE
How can i speed up number spinning without increase iteration count?

### DISCUSSION THREAD

**Discussion by yulafezmesi** [barvian/number-flow#57] [Help] [answered]
hey, thank you for great package!

wanted to do achieve something like this. i played things timing properties but couldn't make it. could you please help?
 
https://github.com/user-attachments/assets/f09b46ab-cf0d-49f8-81a3-98abe74dd3fc

**Comment by barvian** [comment, marked as answer]
Hi 👋, would something [like this](https://codesandbox.io/p/sandbox/number-flow-react-forked-jx2hxp?workspaceId=cbc394d8-76be-4eee-b6ad-557eaf0d663f) work?

**Reply by yulafezmesi** [reply]
that works, thanks! 

also how can i start animation immediately on first render?  @barvian

**Reply by barvian** [reply]
Hi, see #37 for an example
